homeassistant/components/climate/mqtt.py

```diff
@@ -619,3 +631,23 @@ def supported_features(self):
             support |= SUPPORT_AUX_HEAT
 
         return support
+
+    @property
+    def min_temp(self):
+        """Return the minimum temperature."""
+        # pylint: disable=no-member
+        if self._min_temp:
+            return self._min_temp
```

Comment:
So if the parent entity is converting the temperature to the internal `temp_unit`, I think we should do so here too, right?

Reply:
yeah, good point. missed that logic

Reply:
We should probably also fix whatever bugs were introduced in #14721 since I didn't consider that.

Reply:
Already went ahead in #14730 

This is really confusing thanks to the lack of a consistent use of units cross platforms :(

https://github.com/home-assistant/architecture/issues/10